### ISSUE TITLE
[bug] Unable to deploy image

### ISSUE THREAD

**Issue by vpchung** [Sage-Bionetworks-Challenges/cnb-tools#29]
## Error encountered
```
...
ERROR: failed to solve: process "/bin/sh -c pip install --upgrade pip &&     pip install ." did not complete successfully: exit code: 1
Error: buildx failed with: ERROR: failed to solve: process "/bin/sh -c pip install --upgrade pip &&     pip install ." did not complete successfully: exit code: 1
```
[Workflow run for reference.](https://github.com/Sage-Bionetworks-Challenges/cnb-tools/actions/runs/9135683099/job/25201510588)

## Expected behavior
A new version of the [cnb-tools package](https://github.com/Sage-Bionetworks-Challenges/cnb-tools/pkgs/container/cnb-tools) should be available.

**Comment by vpchung** [comment]
After some sleuthing, I suspect that alpine may be _too_ minimal.  Building with `slim` worked locally, so will be pushing this fix.

**Comment by vpchung** [comment]
Confirming that the latest push has fixed the issue - [v0.3.1 now available](https://github.com/Sage-Bionetworks-Challenges/cnb-tools/pkgs/container/cnb-tools)!
